test.py

```diff
@@ -948,6 +948,17 @@ class TopologyTest(PythonTest):
 
     def __init__(self, test_no: int, shortname: str, suite) -> None:
         super().__init__(test_no, shortname, suite)
+        self.server_log_filenames: list[pathlib.Path] = []
+
+    def print_summary(self) -> None:
+        print("Output of {} {}:".format(self.path, " ".join(self.args)))
+        print(read_log(self.log_filename))
+        for index, log_filename in enumerate(self.server_log_filenames):
+            print(f"Server log of the server.{index + 1}: {log_filename}")
```

Comment:
sorry for re-reviewing this so late.

@tchaikov this new index you introduce here is unnecessary and confusing.
We already have multiple numbers to deal with:
- the identifier of the server, which is used inside the log file name, e.g. scylla-537.log
- the IP of the server (which can change mid test)
- and now here you introduce additional `index` which has absolutely no meaning.

You could print the ID of the server but I think it's redundant, since it's already part of the filename.
Why not simply print the list `self.server_log_filenames`?